db/migrate/20220303200215_remove_support_identifier.rb

```diff
@@ -1,5 +1,5 @@
 class RemoveSupportIdentifier < ActiveRecord::Migration[5.2]
   def change
-    remove_column :users, :support_identifier
+    remove_column :users, :support_identifier, :citext
```

Comment:
just checking - we for sure want to remove this? (thought maybe you said we didn't)

Reply:
we do - this was supposed to be an identifier so we could hide the uuid, I believe? It's certainly not being used by anyone - checked with Dante and CS. Attaching relevant conversation.

![Screen Shot 2022-06-01 at 17 06 06](https://user-images.githubusercontent.com/3905516/171509722-ca6c00b8-5ad0-49f3-941e-d33323957fb6.png)